### PR TITLE
ADD test data for edge case testing of formdata anonymization api

### PR DESCRIPTION
**Pull request by JeremiahUy**

### CODE REVIEW

File: src/test/java/no/nav/arxaas/GenerateEdgeCaseData.java [hunk 1]
```diff
@@ -369,11 +368,104 @@ public static MockMultipartFile testMetaDataWrongFormat() {
...
 
+    public static MockMultipartFile testMetaData3Quasi() {
+        return new MockMultipartFile("metadata", "","application/json", testFormData_metadata_3quasi().getBytes());
+    }
+
+    public static MockMultipartFile testMetaDataMissingPrivacyModel() {
+        return new MockMultipartFile("metadata", "","application/json", testFormData_metadata_missing_privacy_models().getBytes());
+    }
+
+    public static MockMultipartFile testMetaDataToMuchPrivacyModel() {
```
Comment (sonhal) [inline src/test/java/no/nav/arxaas/GenerateEdgeCaseData.java]: To "Many" Privacy models?

File: src/test/java/no/nav/arxaas/GenerateEdgeCaseData.java [hunk 2]
```diff
@@ -369,11 +368,104 @@ public static MockMultipartFile testMetaDataWrongFormat() {
...
     private static String testFormData_metadata_2quasi_wrong_format(){
         return "{\"attributes\":[{\"field\":\"aGe\",\"attributeTypeModel\":\"IDENTIFYING\",\"hierarchy\":null}," +
                 "{\"field\":\"GendEr\",\"attributeTypeModel\":\"QUASIIDENTIFYING\",\"hierarchy\":null}," +
                 "{\"field\":\"ZiPcoDe\",\"attributeTypeModel\":\"QUASIIDENTIFYING\",\"hierarchy\":1}]," +
                 "\"privacyModels\":[{\"privacyModel\":\"KANONYMITY\",\"params\":{\"k\":5}}]," +
                 "\"suppressionLimit\":0.02}";
     }
-}
+
+    private static String testFormData_metadata_3quasi(){
```
Comment (sonhal) [inline src/test/java/no/nav/arxaas/GenerateEdgeCaseData.java]: Probably better with words not numbers i.e 3 -> three

File: src/test/java/no/nav/arxaas/controller/FormDataAnonymizationEdgeCaseTest.java [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+                .andExpect(status().is(400))
+                .andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8))
+                .andReturn();
+        assertNotNull(result.getResponse());
+        assertNotNull(result.getResponse().getContentAsString());
+    }
+
+    @Test
+    void formdata_anonymization_with_wrong_data_format() throws Exception {
+        MockMultipartFile testDatasetWrongFormat = GenerateEdgeCaseData.testDatasetComma();
```
Comment (sonhal) [inline src/test/java/no/nav/arxaas/controller/FormDataAnonymizationEdgeCaseTest.java:152]: Should this fail?

**Review by sonhal — DISMISSED** [review]

File: src/test/java/no/nav/arxaas/GenerateEdgeCaseData.java (hunk 2 above)
Comment (JeremiahUy) [reply src/test/java/no/nav/arxaas/GenerateEdgeCaseData.java]: solved

File: src/test/java/no/nav/arxaas/controller/FormDataAnonymizationEdgeCaseTest.java (hunk 3 above)
Comment (JeremiahUy) [reply src/test/java/no/nav/arxaas/controller/FormDataAnonymizationEdgeCaseTest.java:152]: solved

File: src/test/java/no/nav/arxaas/GenerateEdgeCaseData.java (hunk 1 above)
Comment (JeremiahUy) [reply src/test/java/no/nav/arxaas/GenerateEdgeCaseData.java]: solved

**Review by andregroseth — APPROVED** [review]
